.github/workflows/check.yml

```diff
@@ -4,7 +4,7 @@ on: [push, pull_request]
 
 env:
   DEBIAN_FRONTEND: noninteractive
-  TESTS_TIMEOUT: 5 # in minutes
+  TESTS_TIMEOUT: 15 # in minutes
```

Comment:
To reduce runtime we could probably reduce the number of parallel tests in `test-document-fuse.sh`, when `TEST_IN_CI` is set.

Reply:
I've done it, as it may just take too much time (up to 20 minutes in CI) to run it in normal mode, we can maybe add scheduled jobs that run this with normal timeouts?